spec.md

```diff
@@ -987,7 +987,7 @@ parentheses, spaces, and braces.
 <?php
 
 foreach ($iterable as $key => $value) {
-    // foreach body
+    // Foreach body
```

Comment:
Capitalized `Foreach` is jarring to me. Why are we specifying the structure anyway when it's already clear from the preceding line of code? It'd be better for the comment to just be `// Body` or `// ...` in my opinion.

Reply:
I guess this comes down to personal preference. I have no strong feelings either way, I simply prefer consistency because inconsistency is jarring to me.

I created an alterantive to this PR: #79